### PR TITLE
homepage layout facelift

### PR DESCRIPTION
**Pull request by zilahir**
i have made some layout modifications on the `homepage`.

My aim to make the feel and behaviour a little more modern, and clean.

Though this PR is related only to the `homepage`, planning to do some facelifts on other parts of the site as well.

### CODE REVIEW

**Comment by zilahir** [comment]
For some reason the `travis` tests  are failing with `ci`.  

```
Test Suites: 2 failed, 191 passed, 193 total
Tests:       2 failed, 1684 passed, 1686 total
Snapshots:   2 failed, 42 passed, 44 total
```

However,  they are passing on `localhost`:  

```
Test Suites: 193 passed, 193 total
Tests:       1686 passed, 1686 total
Snapshots:   44 passed, 44 total
Time:        69.681s, estimated 148s
```

It's also kind of weird that there are `2` extra tests for everything,  and seems like that 2 is the exact ones are failing.

Any idea what is happening with this on the background?

**Comment by midori0507** [comment]
> For some reason the `travis` tests are failing with `ci`.
> 
> ```
> Test Suites: 2 failed, 191 passed, 193 total
> Tests:       2 failed, 1684 passed, 1686 total
> Snapshots:   2 failed, 42 passed, 44 total
> ```
> 
> However, they are passing on `localhost`:
> 
> ```
> Test Suites: 193 passed, 193 total
> Tests:       1686 passed, 1686 total
> Snapshots:   44 passed, 44 total
> Time:        69.681s, estimated 148s
> ```
> 
> It's also kind of weird that there are `2` extra tests for everything, and seems like that 2 is the exact ones are failing.
> 
> Any idea what is happening with this on the background?

Hi, they are just snapshot tests, can be fixed by `yarn test -u`

File: app/assets/styles/_application-variables.scss [hunk 1]
```diff
@@ -67,3 +67,9 @@ $button-horizontal-padding:    15px;
 $app-ResourceCard-height: 180px;
 $app-SearchPage-max-width: 800px;
 $app-SearchPage-margin: 10px;
+
+/* BREAKPOINTS */
+$screen-sm-min: 576px;
```
Comment (midori0507) [inline app/assets/styles/_application-variables.scss]: We use bootstrap grid system. Can you explain the reason behind extra those breakpoints?
In the future, varaamo will use something call Helsinki Design System, that including universal layout for all Helsinki application.

TLDR: this is quite unneccesary
